doc/quickstart/configure.rst

```diff
@@ -479,58 +470,157 @@ speed with which the data can be read from and/or written to disk.
 
 .. _config-dask-default-scheduler:
 
-Dask default scheduler configuration
-------------------------------------
+Custom Dask default scheduler configuration
+-------------------------------------------
 
 The Dask default scheduler can be a good choice for recipes using a small
```

Comment:
```suggestion
The Dask threaded scheduler can be a good choice for recipes using a small
```

Reply:
It would be nice to reduce the use of the word "default" since it does not convey much meaning and means different things in different contexts.